### PR TITLE
Ft contact:Setting up the contact form

### PR DESCRIPTION
**Pull request by kemigabocatherine**
**In this feature;**

-  Add a section for collecting users' contact information.

-  Add styles to the contact area.

- Add an image for the background

### CODE REVIEW

File: top.html [hunk 1]
```diff
@@ -166,6 +166,24 @@ <h2 class="heading">Glad to see<br>you!</h2>
                 <hr>
```
Comment (carlos-ssh) [inline top.html:166]: - Add a `License` in your `README` file.

File: top.html [hunk 2]
```diff
@@ -166,6 +166,24 @@ <h2 class="heading">Glad to see<br>you!</h2>
                 <hr> 
             </ul>
         </section>
+        <section class="Contact">
+            <h1 class="contact-heading">Contact me</h1>
+            <p class="contact-message">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/{form_id}" method="post">
```
Comment (carlos-ssh) [inline top.html:172]: - Your `POST`  method is missing the `{form-id}` Kindly add it.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @kemigabocatherine , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: top.html [hunk 3]
```diff
@@ -166,6 +166,24 @@ <h2 class="heading">Glad to see<br>you!</h2>
                 <hr> 
             </ul>
         </section>
+        <section class="Contact">
+            <h1 class="contact-heading">Contact me</h1>
+            <p class="contact-message">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/{form_id}" method="post/{form_id}">
```
Comment (VanessaAoki) [inline top.html:172]: - [ ] I think you might have misunderstood what the previous code reviewer meant. In order for the formspree to work, you should [register](https://formspree.io/register) on their website. After that you are going to need to follow these steps:

1 - Create a form:
image

2 - Get the provided endpoint:
image
 
3 - Then you should edit your form to be something like this:
```
<form
  action="https://formspree.io/f/**THE_NEW_ENDPOINT**"
  method="POST"
>
```

Hope this makes it easier to understand and implement!

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @kemigabocatherine, 

Great work on making the changes requested by the previous reviewer, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._



really close

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @kemigabocatherine , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**